OpenDreamClient/Rendering/DreamViewOverlay.cs

```diff
@@ -120,7 +125,13 @@ sealed class DreamViewOverlay : Overlay {
             return;
         Box2 screenArea = Box2.CenteredAround(eyeTransform.WorldPosition, args.WorldAABB.Size);
 
-        var entities = _lookupSystem.GetEntitiesIntersecting(args.MapId, screenArea.Scale(1.2f)); //the scaling is to attempt to prevent pop-in, by rendering sprites that are *just* offscreen
+        HashSet<EntityUid> entities;
+        using (_prof.Group("lookup")) {
+            //TODO use a sprite tree.
```

Comment:
I assume rendering performance isn't a big concern atm, but eventually this should use a proper `DMISpriteComponent` lookup tree instead of doing a general entity lookup. I just did a basic test, and of all the entities returned only ~ 60% actually had a `DMISpriteComponent`.

Reply:
It used to be that every entity had a `DMISpriteComponent`, but now that the client creates entities for opaque turfs that would be a good idea.